--- fsarchiverMain.ui	2011-05-18 16:08:33.000000000 +0400
+++ fsarchiverMain_ru.ui	2011-05-18 17:42:10.000000000 +0400
@@ -198,7 +198,7 @@
     </property>
     <widget class="QLabel" name="label_Partition">
      <property name="text">
-      <string>Vorhandene Partitionen:</string>
+      <string>Найденные разделы:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
@@ -208,7 +208,7 @@
     </widget>
     <widget class="QLabel" name="label_folder">
      <property name="text">
-      <string>Sicherungsverzeichnis:</string>
+      <string>Каталоги сохранения:</string>
      </property>
     </widget>
     <widget class="QTreeView" name="treeView">
@@ -385,21 +385,21 @@
      <item>
       <widget class="QPushButton" name="pushButton_save">
        <property name="text">
-        <string>Partition sichern</string>
+        <string>Сохранить раздел</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_restore">
        <property name="text">
-        <string>Partition wiederherstellen</string>
+        <string>Восстановить раздел</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_end">
        <property name="text">
-        <string>Beenden</string>
+        <string>Выйти</string>
        </property>
       </widget>
      </item>
@@ -437,7 +437,7 @@
    </property>
    <widget class="QMenu" name="menuAktionen">
     <property name="title">
-     <string>Aktionen</string>
+     <string>Функции</string>
     </property>
     <addaction name="separator"/>
     <addaction name="action_partition_save"/>
@@ -453,7 +453,7 @@
    </widget>
    <widget class="QMenu" name="menu_Einstellungen">
     <property name="title">
-     <string>Einstellungen</string>
+     <string>Настройки</string>
     </property>
     <addaction name="action_Einstellungen"/>
    </widget>
@@ -485,7 +485,7 @@
      <normaloff>images/disk.png</normaloff>images/disk.png</iconset>
    </property>
    <property name="text">
-    <string>Partition sichern</string>
+    <string>Сохранение раздела</string>
    </property>
   </action>
   <action name="action_partition_restore">
@@ -494,7 +494,7 @@
      <normaloff>images/book_previous.png</normaloff>images/book_previous.png</iconset>
    </property>
    <property name="text">
-    <string>Partition wiederherstellen</string>
+    <string>Восстановление раздела</string>
    </property>
   </action>
   <action name="action_Einstellungen">
@@ -503,7 +503,7 @@
      <normaloff>images/einstellung.png</normaloff>images/einstellung.png</iconset>
    </property>
    <property name="text">
-    <string>Basiseinstellungen</string>
+    <string>Основные настройки</string>
    </property>
   </action>
   <action name="action_Information">
@@ -530,7 +530,7 @@
      <normaloff>images/mbr_restore.png</normaloff>images/mbr_restore.png</iconset>
    </property>
    <property name="text">
-    <string>MBR wiederherstellen</string>
+    <string>Восстановление МБР</string>
    </property>
   </action>
   <action name="action_dir_restore">
@@ -539,7 +539,7 @@
      <normaloff>images/folder_back.png</normaloff>images/folder_back.png</iconset>
    </property>
    <property name="text">
-    <string>Verzeichnis zurückschreiben</string>
+    <string>Восстановление раздела</string>
    </property>
   </action>
   <action name="action_dir_save">
@@ -548,7 +548,7 @@
      <normaloff>images/folder_go.png</normaloff>images/folder_go.png</iconset>
    </property>
    <property name="text">
-    <string>Verzeichnis sichern</string>
+    <string>Сохранение раздела</string>
    </property>
   </action>
   <action name="action_Beenden">
@@ -566,7 +566,7 @@
      <normaloff>../images/applications-internet.png</normaloff>../images/applications-internet.png</iconset>
    </property>
    <property name="text">
-    <string>Partition sichern Netzwerk</string>
+    <string>Сохранение раздела по сети</string>
    </property>
   </action>
   <action name="action_partition_restore_net">
@@ -575,7 +575,7 @@
      <normaloff>../images/applications-internet-ein.png</normaloff>../images/applications-internet-ein.png</iconset>
    </property>
    <property name="text">
-    <string>Partition wiederherstellen Netzwerk</string>
+    <string>Восстановление раздела по сети</string>
    </property>
   </action>
  </widget>
